screens/cart.html

```diff
@@ -0,0 +1,213 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="../style.css">
+    <link rel="stylesheet" href="../stylesheets/screens/productslist.css">
+    <title>Cart | ArtStation</title>
```

Comment:
Can this second stylesheet be imported to the main style.css?

Reply:
made changes to include only one main css